app/components/card/CardWithMultipleContent.tsx

```diff
@@ -1,64 +1,71 @@
-import type { Content } from "@prisma/client";
+import type { Content, Tag } from "@prisma/client";
 import SecondaryButton from "../buttons/SecondaryButton";
 import Card from "./Card";
-
+import type { SerializeFrom } from "@remix-run/node";
+import { NavLink } from "@remix-run/react";
 interface CardWithMultipleContentProps {
```

Comment:
Legge til litt space her, for bedre lesbarhet?

Reply:
Ja, lurt!